.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
This file should be deleted. Since we added it in the .gitignore fiel it also won't be committed automatically in the future

Reply:
deleted